website/docs/reference/warehouse-profiles/trino-profile.md

```diff
@@ -0,0 +1,63 @@
+---
+title: "Trino Profile"
+---
+
+:::info Community plugin
+
+Some core functionality may be limited. If you're interested in contributing, check out the source code for each repository listed below.
+
+:::
+
+## Overview of dbt-trino
+**Maintained by:** Community    
+**Author:** findinpath    
+**Source:** https://github.com/findinpath/dbt-trino    
+**Core version:** v0.20.0 and newer     
+
+![dbt-presto stars](https://img.shields.io/github/stars/findinpath/dbt-trino?style=for-the-badge)
+
+## Set up a Trino Target
+
+Trino targets should be set up using the following configuration in your `profiles.yml` file.
+
+<File name='~/.dbt/profiles.yml'>
+
+```yaml
+trino:
+  target: dev
+  outputs:
+    dev:
+      type: trino
+      method: none  # optional, one of {none | ldap | kerberos}
+      user: [user]
+      password: [password]  # required if method is ldap or kerberos
+      database: [database name]
+      host: [hostname]
+      port: [port number]
+      schema: [your dbt schema]
+      threads: [1 or more]
+
+```
+
+</File>
+
+## Installation and Distribution
```

Comment:
Could you swap the section about Installation and the Profile?
This way it follows a natural progression from installation to setting up the profile